code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -29,6 +29,7 @@
 	//var/list/viruses = list()
 	var/color = "#000000" //rgb: 0, 0, 0 (does not support alpha channels - yet!)
 	var/alpha = 255
+	var/dupeable = TRUE	//whether the reagent can be duplicated by standard reagent duplication methods such as a service borg shaker or odysseus
```

Comment:
honestly if Shadowmech88 feels like it I'd take it of course but I don't think it's necessary in this case